packages/block-editor/src/components/block-list/style.scss

```diff
@@ -411,6 +411,11 @@
 		}
 	}
 
+	// Center
+	&[data-align="center"] {
+		clear: both;
+	}
```

Comment:
I still have the same question, not sure if it was answered. This behavior only applies to the editor, do we want to apply the same behavior in the frontend too?

Reply:
it makes sense to have the same behavior right?

Reply:
It does for me 🤷‍♂ but maybe we're expecting the theme to define that?